### PR TITLE
added rhsso-satellite integration support test

### PR DESCRIPTION
**Pull request by omkarkhatavkar**
New PR from https://github.com/SatelliteQE/robottelo/pull/7657

### PR objective
To Support the integration login test for RH-SSO and Satellite. 
  
### Test Result

```
/home/okhatavk/okhatavk/Satellite/robottelo/env/bin/python /home/okhatavk/Downloads/pycharm-community-2018.1.4/helpers/pycharm/_jb_pytest_runner.py --target test_ldap_authentication.py::test_single_sign_on_using_rhsso
Launching py.test with arguments test_ldap_authentication.py::test_single_sign_on_using_rhsso in /home/okhatavk/Satellite/robottelo/tests/foreman/ui

============================= test session starts ==============================
platform linux -- Python 3.6.8, pytest-4.6.3, py-1.8.1, pluggy-0.13.1
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/okhatavk/Satellite/robottelo

                                            [100%]

========================== 1 passed in 167.12 seconds ==========================
```

#### Airgun PR 
https://github.com/SatelliteQE/airgun/pull/457

### CODE REVIEW

File: tests/foreman/ui/test_ldap_authentication.py [hunk 1]
```diff
@@ -225,15 +230,111 @@ def enroll_idm_and_configure_external_auth():
         cmd='ipa service-add HTTP/{}'.format(settings.server.hostname),
         hostname=settings.ipa.hostname_ipa,
     )
-    _desired_pos = settings.ipa.hostname_ipa.find('satqe')
-    domain = settings.ipa.hostname_ipa[_desired_pos:]
+    domain = settings.ipa.hostname_ipa[settings.ipa.hostname_ipa.find('satqe') :]  # noqa
```
Comment (mirekdlugosz) [inline tests/foreman/ui/test_ldap_authentication.py]: This is better (shorter, without `# noqa`) written with split (at least as long as FQDN here doesn't change too much):
```python
_, domain = settings.ipa.hostname_ipa.split('.', 1)
```
Proof:
```python
>>> _, domain = settings.ipa.hostname_ipa.split('.', 1)
>>> domain == settings.ipa.hostname_ipa[settings.ipa.hostname_ipa.find('satqe') :]
True
```

**Comment by omkarkhatavkar** [comment]
Travis failure is unrelated to this PR, it is failing because of make docs.

**Comment by omkarkhatavkar** [comment]
Can we please merge this ? 313 gitlab PR is already merged.

**Comment by vijay8451** [comment]
Airgun PR is merged hence merging this one now.

**Pull request merged**